managing_providers/_topics/overview_ibm_cloud_vpc.md

```diff
@@ -0,0 +1,3 @@
+# IBM Cloud Virtual Server for VPC 
+
+[IBM Cloud Virtual Server for VPC](https://www.ibm.com/cloud/vpc) (virtual private cloud) consist of virtual machines (VMs) within IBM’s VPC infrastructure, a public cloud construct. Virtual servers for VPC enable the deployment of a secured private space in a publicly accessible environment by way of a single-tenant infrastructure in a multitenant public cloud. VPCs are logically isolated networks where cloud compute, storage and networking resources are deployed and used to host business workloads, applications and websites. Like networks in traditional on-premises data centers, VPCs utilize cloud security functions and models that match the functionality of those found in offline data centers.
```

Comment:
A couple minor comments:
- "consist of virtual machines" should be "consists"
- "multitenant" should be "multi-tenant"
- First sentence mentions VMs, while second sentence mentions "virtual servers" (which is the proper IBM Cloud VPC term).  I would suggest using the "virtual server" or "virtual server instance" (VSI) term.  Especially since within MIQ, the VSIs are tracked as Cloud "Instances" not "VMs".

Reply:
@chadh1313 thank you.  I have updated the doc based on your recommendations.